packages/expo-updates/android/src/main/java/expo/modules/updates/manifest/ManifestResponse.java

```diff
@@ -14,7 +15,13 @@ public ManifestResponse(Response response) {
     mResponse = response;
   }
 
+  @Nullable
   public String header(String name) {
-    return mResponse.header(name);
+    return this.header(name,null);
```

Comment:
That's fair, I was thinking to keep them intentionally coupled as they are overloaded functions.

Reply:
Your thinking makes sense and would be what I would do in almost any other instance of overloaded functions -- this is just a weird special case because this particular class only exists to wrap okhttp3.Response as transparently/dumbly as possible.